Reduce connection time for new darknode (#71)
renproject/lightnode: CHANGELOG.md

## 0.1.10

> - Improve p2p logic of the lightnode which will reduce connection time for new darknode.

Reviewer: ```suggestion
- Improve P2P logic to reduce connection times for new Darknodes
```
